src/main/java/net/floodlightcontroller/staticflowentry/StaticFlowEntryPusher.java

```diff
@@ -365,19 +365,21 @@ void parseRow(Map<String, Object> row, Map<String, Map<String, OFFlowMod>> entri
 				if (row.get(key) == null) {
 					continue;
 				}
+				
 				if (key.equals(COLUMN_SWITCH) || key.equals(COLUMN_NAME) || key.equals("id")) {
 					continue; // already handled
 				}
-				// explicitly ignore timeouts and wildcards
-				if (key.equals(COLUMN_HARD_TIMEOUT) || key.equals(COLUMN_IDLE_TIMEOUT)) {
```

Comment:
So the notion of the static flow pusher is that it pushes static STATIC flows. If you respect the hard/idle timeout values flows can expire, but will still be displayed by the SFP database which is inconsistent. Make sense? @rizard